mmv1/third_party/terraform/.teamcity/components/inputs/services_beta.kt

```diff
@@ -616,6 +616,11 @@ var ServicesListBeta = mapOf(
         "displayName" to "Secretmanager",
         "path" to "./google-beta/services/secretmanager"
     ),
+    "secretmanagerregional" to mapOf(
+        "name" to "secretmanagerregional",
+        "displayName" to "Secretmanagerregional",
```

Comment:
Why is this needed?
We are setting displayName = Secrets Manager in products.yaml, is this an override.
I think this is the name used in the terrafrom registry to list the products, we want both regional and non-regional resources to be listed under the same listing.
https://github.com/GoogleCloudPlatform/magic-modules/pull/11678/files#diff-c1cff9a2fbc2183ca2638f5cc05943ff90a42751de6dad07cece718f78bf9fe3R16

Reply:
Verified, existing resources like IAM are also adding this.